### ISSUE TITLE
The original repo "Types-for-Adobe" has a new API Version (2022). How can I use it with this extension?

### ISSUE THREAD

**Issue by futuremotiondev** [Afanyiyu/vscode-types-for-adobe#5]
I want to use the Illustrator 2022 set of types found here: 
https://github.com/aenhancers/Types-for-Adobe/tree/master/Illustrator

Is it easy to do this manually? Would you mind adding the updated types to this extension?

Thanks,
Jay

**Comment by yozhic** [comment]
> Is it easy to do this manually?

Yeah, it's quite easy to do. Just clone the _Types-for-Adobe_ repo, copy all the files (or just the ones you need) and replace the existing ones (don't forget the `shared` folder) into the extensions folder:
`~/.vscode/extensions/il-harper.vscode-types-for-adobe-0.1.1/node_modules/types-for-adobe/`

And then edit file
`~/.vscode/extensions/il-harper.vscode-types-for-adobe-0.1.1/out/productInfo.js`
